### ISSUE TITLE
Find more efficient way to rotate credentials that are not owned by Galasa

### ISSUE THREAD

**Issue by jadecarino** [galasa-dev/projectmanagement#1318]

**Comment by jadecarino** [comment]
@hobbit1983 Hi Will, can I close this item now that we have removed all non-Galasa owned credentials and have put our own into the creds? Or do our new credentials also need rotating every 90 days? If so, I'll just rename this item to 'Find automated way to rotate credentials in CREDS'

**Comment by hobbit1983** [comment]
Yes I think we need a way of automating the credential rotation and re-applying it

**Comment by jadecarino** [comment]
@hobbit1983 Thanks, okay I have updated the issue to reflect what we should now do
